tests/test_create_pod_attaching_multiple_pvc_start_IO.py

```diff
@@ -0,0 +1,376 @@
+    log.info(f"Create secret for RBD storageclass")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template, **secret_data
+    )
+    secret_name = data['metadata']['name']
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    assert SEC.create(yaml_file=TEMP_YAML)
+    open(TEMP_YAML, 'w').close()
+    sleep(5)
+    try:
+        # Check for secret creation success from oc
+        SEC.get(secret_name)
+        log.info(f"Secret {secret_name} creation success")
+        return True
+    except exceptions.CommandFailed:
+        assert log.info(f'Exception Secret {secret_name} does not exists ')
+        return False
+
+
+def create_rbd_storage_class():
+    """
+    Create Storage class in oc
+
+    """
+    template = os.path.join(TEMPLATES_DIR, "storageclass.yaml")
+    log.info(f"Create rbd csi storageclass")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template
+    )
+    storage_class_name = data['metadata']['name']
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    assert SC.create(yaml_file=TEMP_YAML)
+    open(TEMP_YAML, 'w').close()
+    sleep(5)
+    try:
+        # Check rbd storageclass creation success from OC
+        SC.get(storage_class_name)
+        log.info(f"SC {storage_class_name} creation success")
+        return True
+    except exceptions.CommandFailed:
+        log.info(f'Exception SC {storage_class_name} does not exists ')
+        return False
+
+
+def create_persistent_volume(volName):
+    """
+    Create a new Persistent Volume
+    """
+    rbd_pvc_data = {}
+    rbd_pvc_data['rbd_pvc_name'] = volName
+    log.info(f"rbd_pvc_data {rbd_pvc_data}")
+    template = os.path.join(TEMPLATES_DIR, "pvc.yaml")
+    log.info(f"Create Persistent Volume Claim ")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        template, **rbd_pvc_data
+    )
+    with open(TEMP_YAML, 'w') as yaml_file:
+        yaml.dump(data, yaml_file)
+    assert PVC.create(yaml_file=TEMP_YAML)
+    open(TEMP_YAML, 'w').close()
+    # Check for pv creation succeed from oc
+    try:
+        PVC.wait_for_resource(
+            resource_name=volName, condition='Bound', timeout=20
+        )
+        log.info(f"PVC {volName} creation success")
+        return True
+    except exceptions.CommandFailed:
+        log.info(f"Exception PVC {volName} not Bound")
+        return False
+
+
+def create_pod(pv_list):
+    """
+    Generate cirros yaml with multiple pvc and run IO's
+
+    """
+    yaml_data = os.path.join(CIRROS_DIR, "cirros.yaml")
+    data = templating.generate_yaml_from_jinja2_template_with_data(
+        yaml_data
+    )
+    cirros_pod_name = data['metadata']['name']
+    temp_dict1 = []
+    temp_dict2 = []
+    temp_str = ''
+    temp_list1 = data['spec']['template']['spec']['containers'][0]['args']
+    temp_list1.pop(len(temp_list1) - 1)
```

Comment:
please add a comment with what you are doing here

Reply:
ack